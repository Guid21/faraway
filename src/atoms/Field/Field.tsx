import { Typography } from 'antd';

const { Text } = Typography;

type FieldProps = Readonly<{
  title: string;
  children: React.ReactNode;
}>;

export const Field = ({ title, children }: FieldProps) => {
  return (
    <div>
      <div>
        <Text type="secondary">{title}</Text>
      </div>
      <div>{children}</div>
    </div>
  );
};