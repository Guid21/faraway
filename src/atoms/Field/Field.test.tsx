import { render } from '@testing-library/react';
import { Field } from './Field';

describe('Field', () => {
  it('renders correctly', () => {
    const { container } = render(<Field title="title">content</Field>);

    expect(container).toMatchSnapshot();
  });
});
