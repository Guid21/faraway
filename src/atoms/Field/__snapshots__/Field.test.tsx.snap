// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field renders correctly 1`] = `
<div>
  <div>
    <div>
      <span
        class="ant-typography ant-typography-secondary css-dev-only-do-not-override-w8mnev"
      >
        title
      </span>
    </div>
    <div>
      content
    </div>
  </div>
</div>
`;
